src/screens/Forum/ForumManagement/AddAdminScreen.js

```diff
@@ -0,0 +1,105 @@
+import { useNavigation } from "@react-navigation/native";
+import { useState } from "react";
+import styled from "styled-components/native";
+import GetUserWithUsername from "../../../services/Friend/GetUserWithUsername";
+import RenderUserToManage from "../../../components/Forum/ForumManagement/RenderUserToManage";
+import { manageType } from "../../../constants/Forum";
+
+const AddAdminScreen = () => {
+  const [username, setUsername] = useState("");
+  const [showUserDetails, setShowUserDetails] = useState(false);
+  const [userData, setUserData] = useState(null);
+
+  function handleChangeText(text) {
+    setUsername(text);
+  }
+
+  console.log(data);
```

Comment:
There is no variable named data, and sometimes this will give a reference error (Though more often it works fine)